pkg/util/util.go

```diff
@@ -29,3 +31,18 @@ func IsGPUVMI(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+// IsIpv6Disabled returns if IPv6 is disabled according sysctl
+func IsIpv6Disabled() bool {
```

Comment:
Might need to be refactor as Vladik suggested on #3396
investigating still why using file read doesn't work on prow, 
and then its possible to move it to new net lib at util as Alona suggested
and there are some more re-factoring possible, that i saw but not related to this PR.
in any case, imo it shouldn't block. i will have more info soon, and act accordingly